src/app/zap-templates/zcl/data-model/chip/thread-network-diagnostics-cluster.xml

```diff
@@ -164,6 +164,11 @@ limitations under the License.
       <description>Indicate that a Node’s connection status to a Thread network has changed</description>
       <field id="0" name="ConnectionStatus" type="ThreadConnectionStatus"/>
     </event>
+    <event side="server" code="0x01" name="NetworkFaultChange" priority="info" optional="true">
```

Comment:
In the spec's Events table for the cluster this is called `NetworkFault`...  @yufengwangca @woody-apple why the difference between spec and SDK here?  Which one is wrong?

Reply:
There is inconsistency in the spec, it is called NetworkFaultChange in another place.
https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/service_device_management/DiagnosticsThread.adoc#91-networkfaultchange-event

9.1. NetworkFaultChange Event

I chose NetworkFaultChange since its field type is also called "NetworkFault"

Reply:
@yufengwangca Did you file a spec issue?